pkg/chains/objects/objects.go

```diff
@@ -0,0 +1,146 @@
+package objects
```

Comment:
Please add godoc comments for the package + types! https://go.dev/doc/comment

Reply:
fixed, feel free to take another look to see if more detail should be added and that the correct formatting is being followed.